src/DependencyInjection/SonataMediaExtension.php

```diff
@@ -99,7 +99,7 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('serializer.xml');
         $loader->load('command.xml');
 
-        if (!in_array(strtolower($config['db_driver']), ['doctrine_orm', 'doctrine_mongodb', 'doctrine_phpcr'])) {
+        if (!in_array(strtolower($config['db_driver']), ['doctrine_orm', 'doctrine_mongodb', 'doctrine_phpcr', 'no_driver'])) {
```

Comment:
can we extract this array a `const` maybe, what do @sonata-project/contributors think?

Reply:
Sounds better, if we don't produce any BC breaks I would prefer that way too

Reply:
ah, forgot about this @covex-nn can you move this without doing a BC break?

Reply:
Sure! i'll do it a little bit later (i must go right now)

Reply:
Blocked merge, thank you for helping!

Reply:
@kunicmarko20 all done, please review )